ding/world_model/utils.py

```diff
@@ -23,3 +28,255 @@ def get_rollout_length_scheduler(cfg: EasyDict) -> Callable[[int], int]:
         return lambda x: cfg.rollout_length
     else:
         raise KeyError("not implemented key: {}".format(cfg.type))
+
+
+def symlog(x):
+    return torch.sign(x) * torch.log(torch.abs(x) + 1.0)
+
+
+def symexp(x):
+    return torch.sign(x) * (torch.exp(torch.abs(x)) - 1.0)
+
+
+class SampleDist:
```

Comment:
Add class notation

Reply:
Why this class is needed? It seems do not contain any useful information?
You can just rewrite these methods into single line code.